src/arrayMethodSort.js

```diff
@@ -4,9 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsString) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prev = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(prev, current) > 0) {
+          count++;
+          this[i - 1] = current;
+          this[i] = prev;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+function compareAsString(a, b) {
+  const stringA = '' + a;
+  const stringB = '' + b;
```

Comment:
Also you can use `String()` here:
```suggestion
  const stringA = String(a);
  const stringB =  String(b);
```